docs/code/README.md

```diff
@@ -17,4 +17,42 @@ This document defines semantic conventions for source code.
 > * SHOULD NOT change the version of the `code` conventions that they emit by default
 >   until the `code` semantic conventions are marked stable.
 
+## Attributes
```

Comment:
could you please replace the table we have in general conventions with the link to this doc (so we don't duplicate)?

https://github.com/open-telemetry/semantic-conventions/blob/8873c264fd85fa7db069761822c1a8312a4022a3/docs/general/attributes.md?plain=1#L426-L460

Reply:
Done!